.github/workflows/linux-build-and-test.yaml

```diff
@@ -0,0 +1,143 @@
+name: Regression Linux
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
```

Comment:
Should it be different time than other scheduled workflows, so they don't wait for resources?

Reply:
i think there is value in having them finish roughly at the same time, since this is trivial to change we can change later if it turns out as problem